Sources/MongoSwift/BSON/BSONValue.swift

```diff
@@ -750,6 +750,45 @@ extension Int64: BSONNumber {
     }
 }
 
+/// A struct to represent the BSON Code type.
+public struct Code: BSONValue, Equatable, Codable, Hashable {
```

Comment:
should `CodeWithScope` be updated to have non-optional scope and always have BSON type `.javascriptWithScope`? or is that coming in future PR?

Reply:
Yep. That'll come in the next PR.